include/s3select.h

```diff
@@ -1218,6 +1226,29 @@ void push_between_filter::builder(s3select* self, const char* a, const char* b)
   self->getAction()->exprQ.push_back(func);
 }
 
+void push_not_between_filter::builder(s3select* self, const char* a, const char* b) const
+{
+  std::string token(a, b);
+
+  std::string not_between_function("#not_between#");
```

Comment:
is "not_between_function" a kind of string constant? if so, it is conventional (and more efficient) to declare it as an actual string (or string_view) constant

Reply:
yes, it could/should be constant.

this method is been called upon parsing only, a few times.

Reply:
static constexpr const char* not_between = "#not_between#";